docs/grammar/SolidityLexer.tokens

```diff
@@ -0,0 +1,272 @@
+ReservedKeywords=1
```

Comment:
Does this file need to be under version control? Isn't it just some intermediate file that can be easily recreated?

Reply:
Nope it doesn't... that was also that wretched wrong push...